### PR TITLE
Breaking changes prep for 3.19

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/release/breaking-changes/multi-touch-scrolling.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Default multitouch scrolling
+title: Default multi-touch scrolling
```
Comment (sfshaza2) [inline src/release/breaking-changes/multi-touch-scrolling.md:2]: Actually, I checked this out very recently. "Multitouch" is becoming more accepted, so I went with no hyphen.

File: src/release/breaking-changes/tooltip-semantics-order.md [hunk 2]
```diff
@@ -25,9 +25,10 @@ As a result, `Tooltip.message` was not immediately after
 ## Migration guide
 
 This change moved the tooltip message in the semantics tree.
-You may see accessibility test failures if your tests expect a tooltip message to
-appear in a specific location in the semantics tree, when it is visible.
-Update your failing accessibility tests to adopt the new tooltip semantics order.
+You might see accessibility test failures if
```
Comment (sfshaza2) [inline src/release/breaking-changes/tooltip-semantics-order.md:28]: Great catch!! may->might

**Review by sfshaza2 — APPROVED** [review]
LGTM Though I take issue with "multi-touch".  :D

File: src/release/breaking-changes/key-event-migration.md [hunk 3]
```diff
@@ -248,58 +248,58 @@ Relevant PRs:
...
-[`KeyUpEvent`]: {{site.main-api}}/flutter/services/KeyUpEvent-class.html
-[`RawKeyEvent.isKeyPressed`]: {{site.main-api}}/flutter/services/RawKeyEvent/isKeyPressed.html
-[`RawKeyEvent.isControlPressed`]: {{site.main-api}}/flutter/services/RawKeyEvent/isControlPressed.html
-[`RawKeyEvent.isShiftPressed`]: {{site.main-api}}/flutter/services/RawKeyEvent/isShiftPressed.html
-[`RawKeyEvent.isAltPressed`]: {{site.main-api}}/flutter/services/RawKeyEvent/isAltPressed.html
-[`RawKeyEvent.isMetaPressed`]: {{site.main-api}}/flutter/services/RawKeyEvent/isMetaPressed.html
-[`RawKeyEvent.repeat`]: {{site.main-api}}/flutter/services/RawKeyEvent/repeat.html
-[`RawKeyEventDataAndroid.eventSource`]: {{site.main-api}}/flutter/services/RawKeyEventDataAndroid/eventSource.html
-[`KeyEvent.deviceType`]: {{site.main-api}}/flutter/services/KeyEvent/deviceType.html
+[`debugKeyEventSimulatorTransitModeOverride`]: {{site.api}}/flutter/services/debugKeyEventSimulatorTransitModeOverride-class.html
```
[the remark this answers is not in the document]
Comment (parlough) [reply src/release/breaking-changes/key-event-migration.md:251]: They're useful for these breaking change guides and elsewhere, as it allows linking to API docs for new APIs not yet released to `stable`.

But now that these will all be in stable in 3.19, they can use the normal links :)

File: src/release/breaking-changes/multi-touch-scrolling.md (hunk 1 above)
Comment (parlough) [reply src/release/breaking-changes/multi-touch-scrolling.md]: Looks weird to me, but I can learn to accept it :P - Reverted this change.

**Pull request merged**